renovate.json

```
@@ -1,3 +1,42 @@
 {
-  "extends": ["github>ministryofjustice/arn-renovate-config"]
-}
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:node"],
+  "prBodyTemplate": "{{{table}}}{{{notes}}}{{{warnings}}}{{{controls}}}",
+  "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "rangeStrategy": "bump"
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major NPM dependencies",
+      "groupSlug": "all-npm-minor-patch",
+      "stabilityDays": 5
+    },
+    {
+      "matchDepTypes": ["engines"],
+      "enabled": false
+    },
+    {
+      "matchPackageNames": ["typescript", "govuk-frontend"],
+      "rangeStrategy": "bump",
+      "stabilityDays": 0
+    },
```

If this is an attempt to replicate the functionality from the old `check_outdated` CI script, should this specify the update type and force the check? I might be reading it wrong, but it seems like this on its own won't do anything?

```suggestion
    {
      "matchPackageNames": ["typescript", "govuk-frontend"],
      "matchUpdateTypes": ["major"],
      "enabled": true,
      "rangeStrategy": "bump",
      "stabilityDays": 0
    },
```